action.yaml

```diff
@@ -33,9 +33,12 @@ inputs:
 runs:
   using: composite
   steps:
-    - id: main
-      shell: bash
-      run: ${{ github.action_path }}/run.sh deploy
+    - uses: gacts/run-and-post-run@v1
+      with:
+        run: |
+          ${{ github.action_path }}/run.sh deploy
+        post: |
+          ${{ github.action_path }}/run.sh teardown
```

Comment:
Well, I think you shouldn't assume it will be VM based since you can run act locally -- given you just have Docker access, and act pretty much replicates 60% of GHA to let you run it locally so that you can test the workflow before it goes live on master.